api_tests/files/views/test_file_detail.py

```diff
@@ -146,7 +146,7 @@ def test_file_guid_created_with_cookie(
         assert mock_allow.call_count == 1
 
     def test_get_file(self, app, user, file_url, file):
-        res = app.get(file_url, auth=user.auth)
+        res = app.get(f'{file_url}?version=2.2', auth=user.auth)
```

Comment:
This had to be made added because `VersionedDateTimeField` respects the version correctly.